src/status_im/contexts/onboarding/create_profile/view.cljs

```diff
@@ -0,0 +1,101 @@
+(ns status-im.contexts.onboarding.create-profile.view
+  (:require
+    [quo.core :as quo]
+    re-frame.db
```

Comment:
wondering why `re-frame.db` is required

Reply:
Probably some leftover, removed. Thank you